.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "0.11"
+  - "0.10"
+  - iojs
+  - iojs-v1.0.2
+services:
+  - redis-server
+before_install:
+  - "npm config set spin false"
+  - "npm install -g npm@~2"
```

Comment:
Ah, cool. I was just trying to get Travis tests to pass because I'm adding some stuff I want on my npm Enterprise install, and I'd like to have Travis work for my s.